src/Pattern.ts

```diff
@@ -54,7 +52,7 @@ export class Pattern {
   }
 
   set type(value) {
-    console.warn('Setting type has no effect', value)
+    console.warn('Setting type has no effect', value);
```

Comment:
As I said previously. I would prefer this to throw an error.
Think of an app that for some strange reason changes type. That app is now broken and the only indication is the console.